README.md

```diff
@@ -57,6 +57,47 @@ can install the standalone
 [ft-vroom plugin](https://github.com/google/vim-ft-vroom) for older versions of
 vim.
 
+## Vroom cheat sheet
+
+Below is a table of the special symbols and conventions vroom recognizes. See
+the files under [examples/](examples/) and in particular
+[examples/basics.vroom](examples/basics.vroom) for explanations.
+
+<!-- Note for editors: the code spans below use NO-BREAK SPACE characters to
+render literal spaces -->
+| Symbol  | Description     | Action          | Example                   | Controls                 |
+| ------- | --------------- | --------------- | ------------------------- | ------------------------ |
+|         | unindented line | comment         | `This is a comment`       |                          |
+| `  > `  | gt leader       | input           | `  > iHello, world!<ESC>` | `(N.Ns)` (delay)         |
+| `  :`   | colon leader    | command         | `  :echomsg 'A message'`  | `(N.Ns)` (delay)         |
+| `  % `  | percent leader  | text            | `  % Sent to buffer`      | `(N.Ns)` (delay)         |
+| `  `    | 2-space indent  | output (buffer) | `  Compared to buffer`    | `(N)` (buf number)       |
+| `  & `  | ampersand       | output          | `  & :LiteralText`        | `(N)` (buf number)       |
+| `  ~ `  | tilde leader    | message         | `  ~ Echo'd!`             | match modes (verbatim)   |
+| `  |`   | pipe leader     | continuation    | `  |…TO A BIGGER HOUSE!`  |                          |
+| `  ! `  | bang leader     | system          | `  ! echo From Vim`       | match modes (regex)      |
```

Comment:
Fixed.

I was torn about whether to explicitly say "default" explicitly because we're pretty width-constrained. Added the "default" and some `<br>`s to keep the other columns from getting scrunched.